### ISSUE TITLE
Installing Autodock Vina for use in Chimera?

### ISSUE THREAD

**Issue by Knots-hocks** [ccsb-scripps/AutoDock-Vina#94]
I've downloaded the mac os file "vina_1.2.3_mac_x86_64" and put it into a folder on my mac. I've set the Autodock Vina to run as a local file and pointed Chimera to the file I downloaded, but when I try to run it I get an error; "vina_1.2.3_mac_x86_64" is not an executable". Am I misunderstanding the instructions or does Vina 1.2.3 no longer run inside Chimera?

**Comment by diogomart** [comment]
Maybe you need to give it permission to execute:
```
chmod +x vina_1.2.3_mac_x86_64
```

**Comment by Knots-hocks** [comment]
Thanks for this. My mac will now run it (after I overrode the non-verified developer bar), but it fails with a "no output" error. Are there some instructions that show non-specialist users how to run the newer versions of Vina on macs?

**Comment by diogomart** [comment]
We have information on how to run vina from the command line, which identical to vina v1.1.2 for many use cases: https://autodock-vina.readthedocs.io/en/latest/docking_basic.html

1. Is Chimera raising the "no output" error?
2. Does vina print anything when executed directly in the command line?

**Comment by Knots-hocks** [comment]
Yes, the "no output" error occurs in Chimera. Does this mean that the newer versions of Vina cannot be run in Chimera on a mac? Version 1.1.2 and the windows version of 1.2.3, run in Chimera. 

I'm sorry I don't have any experience of running Vina from the terminal on a mac. The instructions you cite say that an additional software suite is also required, ADFR? But there are warning messages about that not running in some versions of mac os? Is there somewhere I find instructions on the syntax etc for use in running Vina from the terminal?

**Comment by diogomart** [comment]
I don't know if the newer version of Vina can run in Chimera.

ADFR has nice tools for receptor preparation, but it is possible to prepare receptors in others ways, with MGLTools or OpenBabel. So it is possible to run Vina without ADFR.

The docs I shared contain the instructions for running Vina from the terminal.

**Comment by rwxayheee** [comment]
Hi, As of today, Current Production Release of UCSF Chimera (1.17) is working with vina_1.2.5_mac_aarch64. 
Closing as stale.
